test/tree_stand/node_test.rb

```diff
@@ -199,8 +199,9 @@ def test_query_for_node_returns_only_matches_within_that_node
       (sum) @sum
     QUERY
 
-    assert_equal(1, matches.size)
-    assert_equal('1 + x * 3', matches.dig(0, 'sum').text)
+    assert_equal(2, matches.size)
```

Comment:
I don't understand why the ++.

Reply:
The test was wrong. It should have returned 2 instead of 1. There are 2 sums in the original source code.